Packs/CrowdStrikeFalconSandbox/Integrations/CrowdstrikeFalconSandboxV2/CrowdStrikeFalconSandboxV2.py

```diff
@@ -0,0 +1,521 @@
+
+
+def underscore_to_space(x: str) -> str:
+    return pascalToSpace(underscoreToCamelCase(x))
+
+
+def crowdstrike_report_state_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    key = get_api_id(args)
+    state = client.get_state(key)
+    return CommandResults(outputs_prefix='CrowdStrike.State', raw_response=state, outputs=state,
+                          readable_output=tableToMarkdown('State', state, headerTransform=underscore_to_space))
+
+
+def crowdstrike_get_environments_command(client: Client, _) -> CommandResults:
+    environments = client.get_environments()
+    environments = [map_dict_keys(env, {'environment_id': 'ID', 'total_virtual_machines': 'VMs_total',
+                                        'analysis_mode': 'analysisMode', 'group_icon': 'groupicon',
+                                        'busy_virtual_machines': 'VMs_busy'}) for env in environments]
+
+    readable_output_column_conversion = {
+        'ID': '_ID', 'description': 'Description', 'architecture': 'Architecture',
+        'VMs_total': 'Total VMS', 'VMs_busy': 'Busy VMS', 'analysisMode': 'Analysis mode',
+        'groupicon': 'Group icon'
+    }
+    return CommandResults(
+        outputs_prefix='CrowdStrike.Environment',
+        outputs_key_field='id',
+        outputs=environments,
+        readable_output=tableToMarkdown('Execution Environments:',
+                                        environments, list(readable_output_column_conversion.keys()), removeNull=True,
+                                        headerTransform=lambda x: readable_output_column_conversion[x]),
+        raw_response=environments
+    )
+
+
+def crowdstrike_get_screenshots_command(client: Client, args: Dict[str, Any]):
+    def to_image_result(image: Dict):
+        return fileResult(image['name'], base64.b64decode(image['image']), entryTypes['entryInfoFile'])
+
+    key = get_api_id(args)
+    ret = [to_image_result(image) for image in client.get_screenshots(key)]
+    return ret if len(ret) > 0 else CommandResults(readable_output='No screenshots returned')
+
+
+def crowdstrike_sample_download_command(client: Client, args: Dict[str, Any]) -> List:
+    hash_value = args['file']
+    response = client.download_sample(hash_value)
+    file_name = response.headers.get('Vx-Filename', hash_value + '.gz')
+
+    command_results = CommandResults(
+        readable_output=f'Requested sample is available for download under the name {file_name}'
+    )
+    return [command_results, fileResult(file_name, data=response.content, file_type=EntryType.FILE)]
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    params: Dict[str, Any] = demisto.params()
+    args: Dict[str, Any] = demisto.args()
+
+    verify_certificate = not params.get('insecure', False)
+
+    proxy = params.get('proxy', False)
+
+    demisto.debug(f'Command being called is {demisto.command()}')
```

Comment:
no need to call `demisto.command()` more than once, lets init it once in the `command` var and use it later